File: trunk/ShadowFrameworkLite20/src/shadow/pipeline/java/SFGL20Function.java
package shadow.pipeline.java;

import java.util.List;

import shadow.pipeline.SFFunction;
import shadow.pipeline.builder.SFExpressionBuilder;
import shadow.pipeline.expression.data.SFExpressionParser;
import shadow.pipeline.parameters.SFParameteri;

public class SFGL20Function extends SFFunction{

	public SFGL20Function(SFParameteri globalV, String function, List<SFParameteri> set) {
		super(globalV, null, set);
		SFExpressionBuilder builder=new SFExpressionBuilder();
		SFExpressionParser.getParser().parseString(function,set,builder);
		this.setFunction(builder.getBuiltExpression());
	}
}

File: ShadowFrameworkJS/src/shadow/pipeline/openGL20/SFGL20Function.js
//Java to JS on 22/03/2012

function SFGL20Function(globalV,func,set){
	this.parameter = globalV;
	
	var builder=new SFExpressionBuilder();
	SFExpressionParser_getParser().parseString(func,set,builder);

	this.setFunction(builder.getBuiltExpression());

}

inherit(SFGL20Function,SFFunction);